compiler/rustc_codegen_ssa/src/mir/intrinsic.rs

```diff
@@ -154,6 +387,16 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                 );
                 return;
             }
+            sym::swap_nonoverlapping_single => {
```

Comment:
This will need an impl for rustc_codegen_cranelift too.

Reply:
@bjorn3 Am I right that cranelift doesn't reuse rustc_codegen_ssa code? I cannot find implementation of `rustc_codegen_ssa::traits::BuilderMethods` there.

Reply:
That's correct; cg_clif doesn't use `cg_ssa`.